### PR TITLE
Post Carousel: Add Design Settings

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/1126

### CODE REVIEW

File: widgets/post-carousel/post-carousel.php [hunk 1]
```diff
@@ -156,6 +156,33 @@ function get_widget_form(){
...
+				'hide' => true,
+				'fields' => array(
+					'navigation_color' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation arrow color', 'so-widgets-bundle' ),
+						'default' => '#fff',
+					),
+					'navigation_color_hover' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation arrow color hover', 'so-widgets-bundle' ),
```
Comment (Misplon) [inline widgets/post-carousel/post-carousel.php:172]: Let's perhaps label this "Navigation arrow hover color".

File: widgets/post-carousel/post-carousel.php [hunk 2]
```diff
@@ -156,6 +156,33 @@ function get_widget_form(){
...
+					),
+					'navigation_background' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation background', 'so-widgets-bundle' ),
+						'default' => '#333',
+					),
+					'navigation_background_hover' => array(
+						'type' => 'color',
+						'label' => __( 'Navigation background hover', 'so-widgets-bundle' ),
```
Comment (Misplon) [inline widgets/post-carousel/post-carousel.php:181]: Let's perhaps label this "Navigation hover background". Unless there is precedence I've missed.

**Comment by Misplon** [comment]
Do you want to add a setting for the overlay hover color?

**Comment by AlexGStapleton** [comment]
How does that look? I've added the overlay hover color setting.

What are your thoughts on moving Design settings above Post Query?

**Comment by Misplon** [comment]
Super, thanks.

I'm ok with the order for now.

Can we bold the Post query tab heading so it matches the Design tab heading?

What do you think of changing the label from Overlay hover color to Thumbnail overlay hover color?

**Comment by AlexGStapleton** [comment]
> Can we bold the Post query tab heading so it matches the Design tab heading?

We can. As that's a general change though (as the Posts Query settings group is always unbolded) I've submitted a PR for this: https://github.com/siteorigin/so-widgets-bundle/pull/1136

**Review by Misplon — APPROVED** [review]

**Review by gregpriday — APPROVED** [review]

**Pull request merged**